### PR TITLE
Build ontop of cpu-docker-compose

### PR DESCRIPTION
**Pull request by pgporada**
- Updated the grafana dashboard to match the picture in examples. This was mistakenly missed by me.
- Added grafana and related configs so it's plug and play with docker-compose.

### CODE REVIEW

**Comment by cpu** [comment]
@pgporada Can you pull `master` into this branch so that CI will pass? Thanks!

File: .travis.yml [hunk 1]
```diff
@@ -24,6 +24,6 @@ before_script:
 script:
   - go vet -mod=vendor ./...
   - go test -mod=vendor -v -race -covermode=atomic -coverprofile=coverage.out -tags=integration ./...
-  - goveralls -coverprofile=coverage.out -service=travis-ci -repotoken $COVERALLS_TOKEN
-  - ct-woodpecker -config ./test/config/ct-woodpecker.localdev.json &
+  - goveralls -coverprofile=coverage.out -service=travis-ci
+  - ct-woodpecker -config ./test/config/config.json &
```
Comment (cpu) [inline .travis.yml]: This diff to the `-config` argument doesn't seem right. https://github.com/letsencrypt/ct-woodpecker/pull/76 renames that file to `ct-woodpecker.localdev.json`.

It causing this branch to fail CI:
> 2018/12/19 20:21:12 Unable to load ct-woodpecker config: open ./test/config/config.json: no such file or directory

**Comment by pgporada** [comment]
> Is setting the providers.options.path to /etc/grafana/provisioning/dashboards all that it took to get Grafana to automatically add the dashboard JSON? Very handy!

Grafana v5 allows someone to place a dashboard json file in a specified path and Grafana will automatically 
ingest it. http://docs.grafana.org/administration/provisioning/

File: docker-compose.yml [hunk 2]
```diff
@@ -79,6 +81,41 @@ services:
...
+      ctnet:
+        ipv4_address: 10.40.50.6
+        aliases:
+          - grafana
+    depends_on:
+      # Grafana pulls metrics from prometheus
+      - prometheus
+
+  grafana-post-config:
+    image: fedora
```
Comment (cpu) [inline docker-compose.yml]: Is this only required to make the Grafana dashboard starred to auto-present post-login?

Pulling in a full Fedora image to run some bash isn't super ideal for people with slow internet (:wave:)

Comment (pgporada) [reply docker-compose.yml]: That's not a problem at all. I can remove it.

File: .travis.yml (hunk 1 above)
Comment (cpu) [reply .travis.yml]: Ahhh! That makes sense. I would probably just fix the diff as an add-on commit in this branch with a message explaining its fixing a merge error.

File: docker-compose.yml (hunk 2 above)
Comment (cpu) [reply docker-compose.yml]: Thanks!

**Comment by cpu** [comment]
@pgporada https://github.com/letsencrypt/ct-woodpecker/pull/76 is merged. Can you catch this branch up with master when you have a chance?

**Comment by cpu** [comment]
I did not notice that this merged into `cpu-docker-compose` and not `master`. Going to coordinate on a do-over PR :'(

**Pull request merged**